frontend/src/app/selectors/experiment.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
```

Comment:
[Working on it!](https://github.com/mozilla/testpilot/pull/1466#issuecomment-250239140)